tbx/work/tests/test_models.py

```diff
@@ -1,15 +1,51 @@
+from django.core.paginator import Page as PaginatorPage
```

Comment:
@engineervix I've muddled through creating these tests for the work pages, based on the one @zerolab made for blogs.